process/smartContract/process.go

```diff
@@ -1052,9 +1054,15 @@ func (sc *scProcessor) ProcessSmartContractResult(scr *smartContractResult.Smart
 	}
 
 	var err error
+	txHash, err := core.CalculateHash(sc.marshalizer, sc.hasher, scr)
```

Comment:
Optional - as above (`computeTransactionHash`).

Reply:
deleted that function